tests/python/driver/tvmc/test_target_options.py

```diff
@@ -60,6 +60,28 @@ def test_mapping_target_args():
     assert reconstruct_target_args(parsed) == {"llvm": {"mcpu": "cortex-m3"}}
 
 
+def test_composite_target_cmd_line_help():
```

Comment:
Since the Vitis AI target is not available on some platforms in CI, you'll need to mark the test with `@requires_vitis_ai`